python/lsst/daf/butler/cli/cliLog.py

```
@@ -152,8 +161,22 @@ def record_factory(*args, **kwargs):
 
         logging.setLogRecordFactory(record_factory)
 
+        # Set up the file logger
+        if log_file:
+            handler = logging.FileHandler(log_file, delay=True)
```

Is `delay` useful? May be better to fail early if file cannot be open? Also there could be non-trivial interaction with potential `chdir` use.

---

Maybe not. I ended up adding this to see if it made a difference when debugging a problem with not being able to close files, but I'll remove it now.